bedrock/firefox/redirects.py

```diff
@@ -231,4 +231,8 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 960543
     redirect(r'^firefox/(?P<vers>[23])\.0/eula', '/legal/eula/firefox-{vers}/'),
+
+    # bug 1224060
+    redirect(r'^ja/firefox/ios/(?P<vers>[0-9]+(\.[0-9]+)*)/(?P<file>releasenotes|system-requirements)',
+             'http://www.mozilla.jp/firefox/ios/{vers}/{file}/'),
```

Comment:
Need to add the argument `locale_prefix=False` to the call to `redirect()` so that your regex will match against the locale. Otherwise I think this is okay. I looked to see if this should be a redirect to `https://www.mozilla.jp` but that gives a cert error, which is very unfortunate. I'd love for that to be fixed and a PR filed here to update all redirects to use TLS.

I'd also like to see a test for this added to `test_redirects/map_globalconf.py`.